README.md

```diff
@@ -47,58 +45,27 @@ When building on Linux it is sometimes necessary to run:
     $ gradle wrapper
     
 Finally, turn on the "Use the new formplayer frontend" feature flag on your CommCareHQ domain
+
+### Contributing
+
+For PRs that just modify code in the Formplayer repo, submit a PR to Formplayer on a separate branch.
+
+
+#### Contributing changes to commcare
+
+Formplayer also has a dependency on the commcare-core repository. The commcare-core `master` branch is not
+stable and Formplayer uses a different branch. The submodule repo `libs/commcare` should always be pointing to
+the `formplayer` branch.
```

Comment:
How do changes in the `formplayer` branch get backported to master? Or do they at all? Seems like we could end up diverging if they don't ever get back into master.